rest-api/dao/dv_order_dao.py

```diff
@@ -120,11 +127,11 @@ def from_client_json(self, resource_json, id_=None, expected_version=None,
         order.id = self.get_id(order)[0]
         order.created = self._get_created_date(participant_id, id_)
         order.version = expected_version
-        order.barcode = resource['barcode']
+        order.barcode = fhir_resource.barcode  # NOTE: not in the FHIR spec
         # @TODO: foreign key to biobank order.biobank order id. implement in DA-953
         # order.biobankOrderId = resource['biobank_order_id']
-        order.biobankStatus = resource['status']
-        order.biobankReceived = parse_date(resource['received'])
+        order.biobankStatus = fhir_resource.status
+        order.biobankReceived = parse_date(fhir_resource.received)  # NOTE: not in the FHIR spec
```

Comment:
This line will behave the same as before this change, but `received` is not present in the example or in the FHIR spec for `SupplyRequest`

Reply:
Thanks for doing this, barcode is internal to our table structure, we infer it from another field, same with received I believe. This will be great for our next dry run. I can tune up things like barcode based on some new request data I have from our partners. This is a huge help. 👍